pkg/apiserver/webhooks/datavolume-validate_test.go

```diff
@@ -245,30 +245,98 @@ var _ = Describe("Validating Webhook", func() {
 			resp := validateAdmissionReview(ar)
 			Expect(resp.Allowed).To(Equal(true))
 		})
+
+		It("should reject DataVolume spec PVC size update", func() {
+			blankSource := cdiv1.DataVolumeSource{
+				Blank: &cdiv1.DataVolumeBlankImage{},
+			}
+			pvc := newPVCSpec(pvcSizeDefault)
+			newDataVolume := newDataVolume("testDv", blankSource, pvc)
+			newBytes, _ := json.Marshal(&newDataVolume)
+
+			oldDataVolume := newDataVolume.DeepCopy()
+			oldDataVolume.Spec.PVC.Resources.Requests["storage"] =
+				*resource.NewQuantity(pvcSizeDefault+1, resource.BinarySI)
+			oldBytes, _ := json.Marshal(oldDataVolume)
+
+			ar := &v1beta1.AdmissionReview{
+				Request: &v1beta1.AdmissionRequest{
+					Operation: v1beta1.Update,
+					Resource: metav1.GroupVersionResource{
+						Group:    cdiv1.SchemeGroupVersion.Group,
+						Version:  cdiv1.SchemeGroupVersion.Version,
+						Resource: "datavolumes",
+					},
+					Object: runtime.RawExtension{
+						Raw: newBytes,
+					},
+					OldObject: runtime.RawExtension{
+						Raw: oldBytes,
+					},
+				},
+			}
+
+			resp := validateAdmissionReview(ar)
+			Expect(resp.Allowed).To(Equal(false))
+		})
+
+		It("should accept DataVolume spec PVC size format update", func() {
```

Comment:
not an objection since this only introduces the test, not change the behaviour:
why do we allow changing the format in a way that changes the size, but not the size?

Reply:
oh, I guess I misread what resource.NewQuantity does.

Reply:
We don't allow changing the size, just format (binary/decima).